packages/core/src/api/apis/definitions/alert.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import ApiRef from '../ApiRef';
+
+export type AlertMessage = {
+  message: string;
+  severity?: 'success' | 'info' | 'warning' | 'error';
```

Comment:
Need to document what the default is here, guess error would make sense? Or we just require it for now and see what ppl end up using.

Reply:
yea there isn't a default since it's optional

Reply:
The current default is success, since we're passing it on to MUI. I guess that is as good as any other value, and will be less surprising to ppl that know MUI. Should still repeat it here though imo.

Reply:
ok update

Reply:
I meant documenting it with a comment here :grin:

Side note is that we likely want to bring out the api doc generation from the internal repo, so comments in API definitions will end up in markdown docs.